account_financial_report_qweb/report/general_ledger.py

```diff
@@ -325,11 +322,11 @@ def _get_account_sub_subquery_sum_amounts(
             """
         sub_subquery_sum_amounts += """
         LEFT JOIN
-            res_currency c ON ml.currency_id = c.id        
+            res_currency c ON a.currency_id = c.id        
```

Comment:
This is what would break for the scenario I described in https://github.com/OCA/account-financial-reporting/pull/390#issuecomment-368967626.
I still think it would be better to consider the currency of each line.

Reply:
I guess that I do not understand the scenario you try to describe from an accounting perspective.

If you expect that all the move lines of an account will have the same secondary currency, you activate the currency in the account, and that's it.

Reply:
I'm sorry but I'm not that good from accounting perspective.
I only thought that it could be possible (at least in Odoo it is, I tried it locally) to have an account without currency, but all its lines (or at least the ones selected in the report) have only one currency.
In this scenario it would be nice to have the totals in that currency.

Reply:
Yes, that is theoretically possible, but you are introducing such complexity to the report, where things are much more simple to manage this way. And also it is consistent with the trial balance.

If you do not hold an account in secondary currency what you are interested is to look into specific transactions held using that currency. For example, I make sure that I purchased to supplier A for $5000, even when my company runs using €. But you are not really interested in the totals of Accounts Payable in $, because some other times you will purchase in other currencies as well.